src/library/librarycontrol.cpp

```diff
@@ -405,16 +405,25 @@ void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
 }
 
 void LibraryControl::setLibraryFocus() {
-    if (m_pSidebarWidget) {
-        // XXX: Set the focus of the library panel directly instead of sending tab from sidebar
-        m_pSidebarWidget->setFocus();
+    // XXX: Set the focus of the library panel directly instead of sending tab from sidebar
```

Comment:
- Was this supposed to be a TODO comment?
- Does this apply to the whole function or only to the invocation of setFocus() (according to the previous placement)?

Reply:
I read it as ToDo for setLibraryFocus()
Can make it explicit.